kayak_font/examples/renderer/extract.rs

```diff
@@ -21,20 +21,23 @@ pub fn extract(
             let layouts = font.get_layout(
                 CoordinateSystem::PositiveYUp,
                 text.horz_alignment,
-                text.position,
-                text.size,
+                (text.position.x, text.position.y),
+                (text.size.x, text.size.y),
                 &text.content,
                 text.line_height,
                 text.font_size,
             );
 
             for layout in layouts {
+                let position = Vec2::new(layout.position.0, layout.position.1);
+                let size = Vec2::new(layout.size.0, layout.size.1);
```

Comment:
```suggestion
                let position = Vec2::from(layout.position);
                let size = Vec2::from(layout.size);
```
[Reference](https://docs.rs/bevy/latest/bevy/prelude/struct.Vec2.html#impl-From%3C(f32%2C%20f32)%3E)